File: app/src/main/java/com/galib/natorepbs2/ui/OfficeHeadFragment.java
package com.galib.natorepbs2.ui;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;

import androidx.databinding.DataBindingUtil;
import androidx.fragment.app.Fragment;
import androidx.lifecycle.LiveData;
import androidx.lifecycle.ViewModelProvider;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.galib.natorepbs2.R;
import com.galib.natorepbs2.utils.Utility;
import com.galib.natorepbs2.databinding.FragmentOfficeHeadBinding;
import com.galib.natorepbs2.db.Employee;
import com.galib.natorepbs2.viewmodel.EmployeeViewModel;
import com.squareup.picasso.Picasso;

public class OfficeHeadFragment extends Fragment {
    LiveData<Employee> officeHead;
    EmployeeViewModel employeeViewModel;
    public OfficeHeadFragment() {

    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        employeeViewModel = new ViewModelProvider(this).get(EmployeeViewModel.class);
        officeHead = employeeViewModel.getOfficeHead();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        FragmentOfficeHeadBinding binding =  DataBindingUtil.inflate(inflater, R.layout.fragment_office_head, container,false);
        binding.setPageTitle(getString(R.string.menu_office_head));
        binding.setFragment(this);
        binding.setLifecycleOwner(getActivity());
        officeHead.observe(getViewLifecycleOwner(), employee -> {
            binding.setEmployee(employee);
            binding.executePendingBindings();
        });
        return binding.getRoot();
    }

    public void onClick(View v, String s){
        int id = v.getId();
        if(id == R.id.callBtn){
            Intent intent = new Intent(Intent.ACTION_DIAL, Uri.parse("tel:" + Utility.bnDigitToEnDigit(s)));
            startActivity(intent);
        } else if( id == R.id.emailBtn){
            Intent intent = new Intent(Intent.ACTION_SENDTO, Uri.parse("mailto:" + s));
            startActivity(intent);
        }
    }
}

File: app/src/main/java/com/galib/natorepbs2/ui/OfficeHeadFragment.kt
package com.galib.natorepbs2.ui

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.databinding.DataBindingUtil
import androidx.fragment.app.Fragment
import androidx.fragment.app.viewModels
import androidx.lifecycle.LiveData
import com.galib.natorepbs2.NPBS2Application
import com.galib.natorepbs2.R
import com.galib.natorepbs2.databinding.FragmentOfficeHeadBinding
import com.galib.natorepbs2.models.Employee
import com.galib.natorepbs2.utils.Utility.Companion.bnDigitToEnDigit
import com.galib.natorepbs2.viewmodel.EmployeeViewModel
import com.galib.natorepbs2.viewmodel.EmployeeViewModelFactory

class OfficeHeadFragment : Fragment() {
    private var officeHead: LiveData<Employee>? = null
    private val employeeViewModel: EmployeeViewModel by viewModels {
        EmployeeViewModelFactory((activity?.application as NPBS2Application).repository)
    }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        officeHead = employeeViewModel.officeHead
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        val binding = DataBindingUtil.inflate<FragmentOfficeHeadBinding>(
            inflater,
            R.layout.fragment_office_head,
            container,
            false
        )
        binding.pageTitle = getString(R.string.menu_office_head)
        binding.fragment = this
        binding.lifecycleOwner = activity
        officeHead!!.observe(viewLifecycleOwner) { employee: Employee? ->
            binding.employee = employee
            binding.executePendingBindings()
        }
        return binding.root
    }

    fun onClick(v: View, s: String) {
        val id = v.id
        if (id == R.id.callBtn) {
            val intent = Intent(Intent.ACTION_DIAL, Uri.parse("tel:" + bnDigitToEnDigit(s)))
            startActivity(intent)
        } else if (id == R.id.emailBtn) {
            val intent = Intent(Intent.ACTION_SENDTO, Uri.parse("mailto:$s"))
            startActivity(intent)
        }
    }
}